src/util.cc

```diff
@@ -218,6 +221,63 @@ int WriteFileSync(v8::Isolate* isolate,
   return WriteFileSync(path, buf);
 }
 
+ssize_t ReadFileSyncRaw(
+    const char* path, size_t path_len, std::string* contents) {
+  // Contains a nul byte.
+  if (strlen(path) != path_len) {
+    return UV_EINVAL;
```

Comment:
```suggestion
    return UV_EINVAL;  // Contains a nul byte.
```
Since we don't know that until control arrives here.